apps/costcontrol/style/views/datausage.css

```diff
@@ -92,27 +104,24 @@
 }
 
 #datausage-tab .app-item {
-  width: 32rem;
   padding: 1.9rem 0 0 0;
-  clear: both;
 }
 
 #datausage-tab .app-item > a {
-  display: block;
+  display: flex;
   overflow: auto;
+  text-decoration: none;
 }
 
 #datausage-tab .app-image {
   padding: 1.1rem 1.3rem 1.1rem 1.3rem;
   width: 3rem;
   height: 3rem;
-  float: left;
 }
 
 #datausage-tab .app-info {
-  width: calc(100% - 11rem);
-  margin-right: 5.4rem;
-  float: right;
+  flex: 1;
+  -moz-margin-end: 5.4rem;
```

Comment:
I suppose the reason of  the margin is maintain an alignment similar to the header section, feel free to ask for a UI review if you think the margin is excessive .